src/creatures/players/player.hpp

```diff
@@ -474,6 +474,10 @@ class Player final : public Creature, public Cylinder, public Bankable {
 	int32_t getStorageValueByName(const std::string &storageName) const;
 	void addStorageValueByName(const std::string &storageName, const int32_t value, const bool isLogin = false);
 
+	std::shared_ptr<KVStore::ScopedKV> kv() {
+		return g_kv().scoped(fmt::format("player.{}", getGUID()));
```

Comment:
Since this is a new global, maybe we could just add it to the constructors of the classes using it instead of introducing a g_*(). Not a big deal though, since we can override the g_* variables in tests, but might be a nice step forward if we want to move fully to constructor injections.